holidays/countries/anguilla.py

```diff
@@ -0,0 +1,157 @@
+        - [Public Holidays For 2013](https://web.archive.org/web/20131113151001/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2014](https://web.archive.org/web/20141113163349/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2015](https://web.archive.org/web/20151113150558/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2016](https://web.archive.org/web/20161021015139/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2017](https://web.archive.org/web/20170830143632/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2018](https://web.archive.org/web/20181127040220/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2019](https://web.archive.org/web/20191021020537/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2020](https://web.archive.org/web/20210205021056/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2021](https://web.archive.org/web/20210901183726/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2022](https://web.archive.org/web/20221011200016/http://www.gov.ai/holiday.php)
+        - [Public Holidays For 2024](https://web.archive.org/web/20241004074741/https://www.gov.ai/service/public-holidays-for-2024)
+        - [Public Holidays For 2025](https://web.archive.org/web/20250425025259/https://www.gov.ai/service/public-holidays-for-2025)
+        - [PUBLIC HOLIDAYS ACT R.S.A. c. P130](https://www.gov.ai/laws/P130-00-Public%20Holidays%20Act/docs/P130-00-Public%20Holidays%20Act_03.pdf)
+        - [Revised Regulations of Anguilla: P130-1](https://www.gov.ai/laws/P130-01-Public%20Holidays%20Regulations/docs/P130-01-Public%20Holidays%20Regulations_05.pdf)
+        - [Government of Anguilla Official Gazette](https://web.archive.org/web/20250304213202/https://gov.ai/document/2025-03-04-123357_2051160063.pdf)
+        - [REVISED REGULATIONS OF ANGUILLA](https://natlex.ilo.org/dyn/natlex2/natlex2/files/download/109059/ATG109059.pdf)
+    """
+
+    country = "AI"
+    default_language = "en_AI"
+    supported_languages = ("en_AI", "en_US")
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    # Declaration of independence May 30, 1967,
+    # but the 2010 revision is the most recent comprehensive legal update.
+    start_year = 2011
+    weekend = {SUN}
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, cls=AnguillaStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("New Year's Day")))
+
+        # James Ronald Webster Day.
+        self._add_observed(self._add_holiday_mar_2(tr("James Ronald Webster Day")))
```

Comment:
Added on [MAR 1, 2010](https://laws.gov.ai/storage/pdfs/2010%20AXA%20Revised%20Statutes%20and%20Regulations/PDF%20(Regulations)/P-R.R.A.s/P130-Public%20Holidays%20Regulations.pdf) (this was the only holiday added by the 2010 amendment)
```suggestion
        if self._year >= 2010:
            # James Ronald Webster Day.
            self._add_observed(self._add_holiday_mar_2(tr("James Ronald Webster Day")))
```